creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -575,6 +607,23 @@ spec:
       networkName: sriov-net-crd
 ----
 
+______________________________________________________________________________
+*Note:* you need to enable the SRIOV feature gate to use the feature. For
```

Comment:
This note is probably a spurious remnant of a rebase. I don't think we document the feature gate anymore considering that it's enabled by default. Please remove the note.